### DISCUSSION TITLE
Any plan to implement gio-qt for QT6?

### DISCUSSION THREAD

**Discussion by karuboniru** [linuxdeepin/developer-center#2865] [Q&A | 问答板块]
Currently we only have gio-qt for QT5, is there any plan to support QT6?

**Comment by BLumia** [comment]
https://github.com/linuxdeepin/gio-qt/pull/18 这里有个 PR 应该可以用，不过可能暂时没法合并进来。我们这边正在着手准备把开发流程迁回 GitHub，届时外部 PR 应该就可以合并进来了。
